tools/cockpit.spec

```diff
@@ -573,15 +623,18 @@ This package contains tests and files used while testing Cockpit.
 These files are not required for running Cockpit.
 
 %files -n cockpit-tests -f tests.list
-%config(noreplace) %{_sysconfdir}/cockpit/cockpit.conf
```

Comment:
Indeed we don't ship a default config file, although we recently [got a request](https://bugzilla.redhat.com/show_bug.cgi?id=1816044) to do so. If that gets in the way, could you please just comment it out instead?

Reply:
that one was a complaint from rpm that a file is packaged twice.

Reply:
How do you mean "twice"? Do you change your cockpit-ws.rpm to include a cockpit.conf already? This one is just for cockpit-tests, which we mostly just use for integration tests; the config causes glib criticals to crash cockpit, so that we notice in our tests if we get a critical. So if you want, you can even completely disable that subpackage.

Reply:
[   81s] ... running 50-check-packaged-twice
  [   81s] found conflict of cockpit-bridge-215-0.x86_64 with cockpit-tests-215-0.x86_64:
  [   81s]   - /etc/cockpit/cockpit.conf

cockpit-bridge has %{_sysconfdir}/cockpit
So the question is what is intended here. Not include the config file in cockpit-bridge means that entry needs a  %dir. If cockpit-bridge is meant to have the file then cockpit-tests doesn't need it as cockpit-tests requires cockpit-bridge.

Reply:
ah, was my fault :/

Reply:
cockpit-bridge absolutely should *not* have cockpit.conf, so if it appears there, that's a packaging bug. That file is configuration for cockpit-ws; however, we don't ship a production default cockpit.conf right now. *This* cockpit.conf for cockpit-tests is really just for integration tests. We can drop it if it gets in the way, and instead create it dynamically in our test suite of course.

Reply:
I sent PR #13837 to eliminate that cockpit.conf file entirely. It will also make life easier on our side anyway. But I'm glad it caught that packaging bug :)